### DISCUSSION TITLE
Syntax error from plugin class

### DISCUSSION THREAD

**Discussion by gdm-1718-indymeer** [swup/swup#682] [Help] [answered]
I cannot make the swup plugins compile. When only using import swup, i have no errors. 

This is the error I get: 
`SyntaxError: Unexpected token: operator (?) while parsing file: /Volumes/WWW/project/node_modules/@swup/plugin/dist/index.cjs
`

This is how I import it in my file: 
`import Swup from 'swup';
import SwupHeadPlugin from '@swup/head-plugin';
import SwupBodyClassPlugin from '@swup/body-class-plugin';
import SwupJsPlugin from '@swup/js-plugin';
import SwupGaPlugin from '@swup/ga-plugin';`

Currently I'm using a gulp stack with babelify to compile my js and css

**Comment by daun** [comment, marked as answer]
Looks like there's some syntax in there that your specific version of babel doesn't (yet) understand. I'd first check if your config includes support for the null-coalescing operator (`a?.b`).
